src/daemons/rhsmcertd.c

```diff
@@ -410,6 +435,13 @@ parse_cli_args (int *argc, char *argv[])
 int
 main (int argc, char *argv[])
 {
+    debug("Getting seed for random number generator from");
```

Comment:
Let's just drop this line since it's always going to get printed regardless of anything else